src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -698,6 +698,15 @@ private void AddCustomFormats(Dictionary<string, Func<TokenMatch, string>> token
             }
 
             tokenHandlers["{Custom Formats}"] = m => string.Join(" ", customFormats.Where(x => x.IncludeCustomFormatWhenRenaming));
+            tokenHandlers["{Custom Format}"] = m =>
+            {
+                if (m.CustomFormat.IsNullOrWhiteSpace())
+                {
+                    return string.Empty;
+                }
+
+                return customFormats.Where(x => x.IncludeCustomFormatWhenRenaming && x.Name == m.CustomFormat).FirstOrDefault()?.ToString() ?? string.Empty;
```

Comment:
This PR only adds the ability to position Custom Format name matches. The ability to use a different word (e.g. WEB-DL instead of the format name "WEB-DL PVP") I held off on because of unanswered questions and needs a separate PR.